containers/javascript-node-lts-postgres/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,37 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+FROM node:lts
+
+# Configure apt
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update \
```

Comment:
We did some RUN layer consolidation around the time you submitted this PR, so could you grab the latest Dockerfile from javascript-node-lts and add it here?

Reply:
Yeah, @Chuxel, I have seen it and yes I have made those change now!